frontend/src/app/projects/project-detail/project-metadata/project-metadata.component.html

```diff
@@ -39,13 +39,26 @@ <h2 id="title">Metadata</h2>
         </mat-form-field>
       </fieldset>
     </form>
-    <button
-      [disabled]="!form.valid"
-      (click)="updateDescription()"
-      mat-raised-button
-      id="submit"
-    >
-      Submit
-    </button>
+    <p *ngIf="!canDelete">
+      The project can't be deleted since it has linked models.
+    </p>
```

Comment:
We already had the discussion how to show the user the reason why he can't click on the delete button in https://github.com/DSD-DBS/capella-collab-manager/pull/444. But I think there was an open point from @MoritzWeber0 and he preferred the tooltip variant (and that's what we are using right now for the model deletion). So I would consistently use one variant and currently also prefer the tooltip variant more.